### DISCUSSION TITLE
my github pages is stuck at Certificate Requested: This domain was recently added.

### DISCUSSION THREAD

**Discussion by amishtp1407** [community/community#36290] [Other Feature Feedback, Questions, & Ideas]
my github pages is stuck at **Certificate Requested: This domain was recently added.** I wanted to host my private repository docusarus website. Website is up, but with not secure https error. Please fix this issue. My organization is halt-dos. Please help me out.

**Comment by yoannchaudet** [comment]
I suggest opening a support request if you still have an issue. Without the name of your repository, I am unable to check its status.

**Comment by ossanna16** [comment]
Hey there @amishtp1407 and thank you for asking a good question! I'm glad you found our community 🙂
